### PR TITLE
Feature/io 571 refactor

### PR DESCRIPTION
**Pull request by elizaveta-seleznyova-idf**
Refactor to the Home Screen 
PLEASE DON'T PAY ATTENTION TO COMMIT COUNT. I can't do anything with it

### CODE REVIEW

**Review by vladimir-hodaryonok-idf — APPROVED** [review]

File: packages/presentation/packages/presentation/lib/lib/generated/intl/messages_en.dart [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// DO NOT EDIT. This is code generated via package:intl/generate_localized.dart
+// This is a library that provides messages for a en locale. All the
+// messages from the main program should be duplicated here with the same
```
Comment (egor-slatvinskiy-idf) [inline packages/presentation/packages/presentation/lib/lib/generated/intl/messages_en.dart]: adding generation files to gitignor

File: packages/presentation/packages/presentation/lib/l10n/intl_en.arb [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (egor-slatvinskiy-idf) [inline packages/presentation/packages/presentation/lib/l10n/intl_en.arb]: why do you need an empty file?

**Review by egor-slatvinskiy-idf — APPROVED** [review]

File: packages/presentation/packages/presentation/lib/l10n/intl_en.arb (hunk 2 above)
Comment (elizaveta-seleznyova-idf) [reply packages/presentation/packages/presentation/lib/l10n/intl_en.arb]: I removed it.Update project

File: packages/presentation/packages/presentation/lib/lib/generated/intl/messages_en.dart (hunk 1 above)
Comment (elizaveta-seleznyova-idf) [reply packages/presentation/packages/presentation/lib/lib/generated/intl/messages_en.dart]: I removed it.Update project

File: packages/presentation/lib/config/theme/app_colors.dart [hunk 3]
```diff
@@ -10,4 +10,5 @@ abstract class AppColorsDark {
   static const unselectedColor = Color.fromRGBO(255, 255, 255, 0.5);
   static const primaryColor = Color.fromRGBO(229, 25, 55, 1);
   static const primaryGradientEnd = Color.fromRGBO(219, 82, 82, 1);
+  static const starColor = Colors.yellow;
```
Comment (yuliya-klescheva-idf) [inline packages/presentation/lib/config/theme/app_colors.dart]: use rgbo for this color

File: packages/presentation/lib/screen/home/widgets/home_shimmer.dart [hunk 4]
```diff
@@ -1,15 +1,19 @@
...
-  const HomeShimmer({super.key}) ;
+  const HomeShimmer({super.key});
 
   @override
   Widget build(BuildContext context) {
     final size = MediaQuery.of(context).size;
     final width = size.width;
+    int itemCountLength = 6;
+    int crossAxisCountSize = 2;
+    int flexSize = 2;
```
Comment (yuliya-klescheva-idf) [inline packages/presentation/lib/screen/home/widgets/home_shimmer.dart]: make it static const if these variables will not change

File: packages/presentation/lib/config/theme/app_colors.dart (hunk 3 above)
Comment (elizaveta-seleznyova-idf) [reply packages/presentation/lib/config/theme/app_colors.dart]: corrected

File: packages/presentation/lib/screen/home/widgets/home_shimmer.dart (hunk 4 above)
Comment (elizaveta-seleznyova-idf) [reply packages/presentation/lib/screen/home/widgets/home_shimmer.dart]: corrected here and in other files

**Review by alexei-kruk-idf — APPROVED** [review]

**Review by yuliya-klescheva-idf — APPROVED** [review]

**Review by sergei-mikhailovskii-idf — APPROVED** [review]

**Pull request merged**